corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -233,9 +242,147 @@ hqDefine("geospatial/js/case_grouping_map",[
         }
     }
 
+    function loadCaseGroupsInExport(caseGroups) {
```

Comment:
Could we rather move this function inside `exportModel` and then call it on the instance itself? It'll keep things more organized having its related function inside the class itself.

Reply:
Yeah, great suggestion!